apps/demo-mfa-nodejs/index.ts

```diff
@@ -0,0 +1,198 @@
+}
+
+if (process.argv.length < 3) {
+  throw Error("Please provide either --lookup or --claim flag");
+}
+
+/**
+ * Set up Lit Node Client
+ */
+
+const litNodeClient = new LitNodeClientNodeJs({
+  litNetwork: "cayenne",
+  debug: false,
+});
+
+await litNodeClient.connect();
+
+const authClient = new LitAuthClient({
+  litRelayConfig: {
+    relayApiKey: LIT_RELAY_API_KEY,
+  },
+  litNodeClient,
+});
+
+/**
+ * Setup STYTCH client
+ */
+
+const client = new stytch.Client({
+  project_id: STYTCH_PROJECT_ID,
+  secret: STYTCH_SECRET,
+});
+
+/**
+ * Email auth with STYTCH
+ */
+const emailResponse = await prompts({
+  type: "text",
+  name: "email",
+  message: "Enter your email:",
+});
+
+const stytchResponse = await client.otps.email.loginOrCreate({
+  email: emailResponse.email,
+});
+
+const otpResponse = await prompts({
+  type: "text",
+  name: "code",
+  message: "Enter the code sent to your email:",
+});
+
+const authResponse = await client.otps.authenticate({
+  method_id: stytchResponse.email_id,
+  code: otpResponse.code,
+  session_duration_minutes: 60 * 24 * 7,
+});
+
+const sessionStatus = await client.sessions.authenticate({
+  session_token: authResponse.session_token,
+});
+
+const session = authClient.initProvider<StytchOtpProvider>(
+  ProviderType.StytchOtp,
+  {
+    userId: sessionStatus.session.user_id,
+    appId: STYTCH_PROJECT_ID,
+  }
+);
+
+authMethod = await session.authenticate({
+  accessToken: sessionStatus.session_jwt,
+});
+
+/**
+ * Phone auth with Stytch
+ */
+const phoneResponse = await prompts({
+  type: "text",
+  name: "phone",
+  message: "Enter your phone number, make sure to include the country code (+1 for US):",
+});
+
+const stytchphoneResponse = await client.otps.sms.loginOrCreate({
+  phone_number: phoneResponse.phone,
+});
+
+const otpPhoneResponse = await prompts({
+  type: "text",
+  name: "code",
+  message: "Enter the code sent to your phone number:",
+});
+
+const authPhoneResponse = await client.otps.authenticate({
+  method_id: stytchphoneResponse.phone_id,
+  code: otpPhoneResponse.code,
+  session_duration_minutes: 60 * 24 * 7,
+});
+
+const sessionPhoneStatus = await client.sessions.authenticate({
+  session_token: authPhoneResponse.session_token,
+});
+
+const phoneSession = authClient.initProvider<StytchOtpProvider>(
+  ProviderType.StytchOtp,
+  {
+    userId: sessionPhoneStatus.session.user_id,
+    appId: STYTCH_PROJECT_ID,
+  }
+);
+
+authPhoneMethod = await phoneSession.authenticate({
+  accessToken: sessionPhoneStatus.session_jwt,
+});
+
+
+/**
+ * Lit Actions code to run after both auth methods have been accepted
+ */
+
+if (process.argv.length >= 3 && process.argv[2] === "--mfa") {
+  
+// this code will be run on the node
+const litActionCode = `
+const go = async () => {
+  Lit.Actions.setResponse({response: JSON.stringify({"Lit.Auth": Lit.Auth})})
+
+  // Check the Lit.Auth object for auth methods (both 7 for OTP auth)
+  if () {
```

Comment:
this is the meat of this PR that never got implemented - the if statement to check that both are present